src/com/android/launcher3/provider/LauncherDbUtils.java

```diff
@@ -171,9 +173,13 @@ public static void migrateLegacyShortcuts(Context context, SQLiteDatabase db) {
                     null);
         }
 
+        // Compat users upgrade from Lawnchair 13.
+        removeColumn(db, Favorites.TABLE_NAME, "iconPackage");
+        removeColumn(db, Favorites.TABLE_NAME, "iconResource");
+
         // Drop the unused columns
-        db.execSQL("ALTER TABLE " + Favorites.TABLE_NAME + " DROP COLUMN iconPackage;");
-        db.execSQL("ALTER TABLE " + Favorites.TABLE_NAME + " DROP COLUMN iconResource;");
+//        db.execSQL("ALTER TABLE " + Favorites.TABLE_NAME + " DROP COLUMN iconPackage;");
+//        db.execSQL("ALTER TABLE " + Favorites.TABLE_NAME + " DROP COLUMN iconResource;");
```

Comment:
Please don't commented this, just surround this with try catch block and `Throwable ` with  
`removeColumn(db, Favorites.TABLE_NAME, "iconPackage");`
`removeColumn(db, Favorites.TABLE_NAME, "iconResource");`